roadmap/entries/data-digital_assets.json

```diff
@@ -2,6 +2,48 @@
   "categoryName": "Digital Assets",
   "categoryDescription": "DeFi protocols, real-word asset tokenization, and other digital asset solutions are strong drivers of web3 adoption. ICP’s Chain Fusion capabilities, unparalleled scalability, and the ability to serve web assets from chain opens up novel digital asset classes. Moreover, it makes ICP an attractive platform to build multi chain custody solutions and wallets.",
   "elements": [
+    {
+      "title": "Solana Integration",
+      "overview": "OISY widens its multi-chain reach beyond Bitcoin and Ethereum, and will integrate with the most important networks. Solana is the first one to be implemented, a crucial step forward in our mission to deliver a true multi-chain wallet for decentralized finance (DeFi) users.",
+      "status": "in_progress",
+      "forum": "",
+      "proposal": "",
+      "docs": "",
+      "is_community": false,
+      "in_beta": false,
+      "milestone_id": "Helios"
+    },
+    {
+      "title": "In-Wallet Swaps",
+      "overview": "Introducing in-wallet swaps, this release will enhance flexibility for managing assets, allowing users to seamlessly change their asset allocation directly in OISY using DEX liquidity, mitigating risk or aiming for a higher yield.",
+      "status": "in_progress",
+      "forum": "",
+      "proposal": "",
+      "docs": "",
+      "is_community": false,
+      "in_beta": false,
+      "milestone_id": "Helios"
+    },
+    {
+      "title": "Promote Chain-Key Tokens",
+      "overview": "With improved screens, flows, and help texts for chain fusion conversions, OISY users will get the right information at the right places to make informed decisions to convert native tokens into their chain fusion twins or vice versa.",
+      "status": "in_progress",
+      "forum": "",
+      "proposal": "",
+      "docs": "",
+      "is_community": false,
+      "in_beta": false,
+      "milestone_id": "Helios"
+    },
+    {
+      "title": "Oisy, a Multi-chain Wallet Powered by Chain Fusion",
```

Comment:
I suggest renaming the milestone title to something more specific mention the Oisy v1 launch.

"Oisy v1 launch" or any variation would work.